jenkins/shared-library/vars/weeklyBuildPipeline.groovy

```diff
@@ -135,11 +135,11 @@ void triggerAllJobs(
 
 		environments.each { environment ->
 			String stageName = "${displayName} (${environment})"
+			String osValue = environment.split('-')[1]
```

Comment:
i thought tags were {operating-system}-{base-image} (from baseImages.yaml) or is this something else?

Reply:
yeah, that's the tag for the base image.  The environment tag includes the environment name, so it would be {environmentName}-{operating-system}-{base-image}.  Something like ``Python-Windows-lts`` or ``Javascript-ubuntu-lts``.
Let me know if you have a better idea

Reply:
I refactor a bit to simplify and hopefully better readability.  Let me know if this is better.